provisioning_templates/snippet/blacklist_kernel_modules.erb

```diff
@@ -0,0 +1,12 @@
+<%#
+kind: snippet
+name: blacklist_kernel_modules
+model: ProvisioningTemplate
+snippet: true
+%>
+
```

Comment:
Can you remove the blank line? This will create blank line on every rendered template, we already have couple and we actually want *less* :-)